### ISSUE TITLE
Missing init_scripts in Cluster Creation?

### ISSUE THREAD

**Issue by jaydm26** [databricks/databricks-sdk-py#121]
Hello,

I believe the current method to create a cluster in the SDK is missing init_scripts (which is a part of the APIs- [Create new cluster](https://docs.databricks.com/api-explorer/workspace/clusters/create)).  I am currently using a custom class which adds the init_scripts part to the body of the request. 

Is adding init_scripts during cluster creation through the SDK planned?

Thank you.

**Comment by nfx** [comment]
@jaydm26 we'll look into why this happening, meanwhile - can you use `w.global_init_scripts`?

**Comment by jaydm26** [comment]
Yeah, I can use global init scripts for the time being. Thank you for looking into it
